Tasks/app-store-release/app-store-release.ts

```diff
@@ -122,7 +123,9 @@ async function run() {
             // To get a FASTLANE_SESSION, run 'fastlane spaceauth -u [email]' interactively (requires PIN)
             // See: https://github.com/fastlane/fastlane/blob/master/spaceship/README.md
             tl.debug('Using two-factor authentication');
-            process.env[fastlaneSessionEnvVar] = credentials.fastlaneSession;
+            if (!appSpecificId) {
```

Comment:
I think it could be unobvious that passing of fastlane session depends on appSpecificId - could we pass it depending on fastlane session value, and print some warnings if other parameters like appSpecificId not set (if fastlane session is not set also)?

Reply:
Fixed